linera-views/src/batch.rs

```diff
@@ -438,4 +435,20 @@ mod tests {
         );
         assert!(simple_unordered_batch.insertions.is_empty());
     }
+
+    #[tokio::test]
+    async fn test_simplify_batch6() {
+        let context = create_memory_context();
+        let insertions = vec![(vec![1, 2, 3],vec![])];
+        let simple_unordered_batch = SimpleUnorderedBatch { insertions: insertions.clone(), deletions: vec![] };
+        let key_prefix_deletions = vec![vec![1,2]];
+        let unordered_batch = UnorderedBatch { simple_unordered_batch, key_prefix_deletions };
+        let unordered_batch_simp = unordered_batch
```

Comment:
nit: `_simp` -> `_simplified` (or better `let expanded_batch = ` ?)

Reply:
That one is outdated.